### ISSUE TITLE
Improve sending functions

### ISSUE THREAD

**Issue by NicoHood** [NicoHood/IRLremote#5]
Sending needs more testing and the bitbanging can be improved with inline assembler.

**Comment by Swap-File** [comment]
Is sending currently supported in V2.0.0?  The examples didn't seem to work so I reverted to 1.7.4 which worked well.

**Comment by NicoHood** [comment]
@Swap-File Nope, not yet supported. I am not happy with the implementation and unless i have a better idea (and time) i cant implement a good solution.
